main.c

```diff
@@ -4,16 +4,6 @@
 #include "lib/dames.h"
```

Comment:
Generally speaking, you would be having, a src dir with all .c, a include dir with .h.

And if needed a directory lib with external libs has .so or sources (even though that's bad practice as project are generally better kept separated.)